cluster/cluster.go

```diff
@@ -202,7 +205,7 @@ func (cluster *Cluster) CheckClusterError(remoteOutput *RemoteOutput, finalErrMs
 			if remoteOutput.Scope != ON_HOSTS {
 				segMsg = fmt.Sprintf("on segment %d ", contentID)
 			}
-			gplog.Verbose("%s %son host %s with error %s: %s", messageFunc(contentID), segMsg, cluster.GetHostForContent(contentID), err, remoteOutput.Stderrs[contentID])
+			gplog.Verbose("%s %son host %s with error %s: %s Command was: %s", messageFunc(contentID), segMsg, cluster.GetHostForContent(contentID), err, remoteOutput.Stderrs[contentID], remoteOutput.CmdStrs[contentID])
```

Comment:
Could you put the "Command was" part in a separate Verbose call, so it's not all in one big line that might wrap two or three times?